rule_schema_v1.yaml

```diff
@@ -473,7 +482,55 @@ $defs:
   id:
     title: Rule ID to attach to findings
     type: string
-type: object
+  languages:
+    title: Languages this pattern should run on
+    type: array
+    items:
+      enum:
```

Comment:
Is there a way to list an enum (so that some tools can leverage the info and provide a popup list in the playground)
but also accept any string? I'm worried we get some errors when we update lang.json and forget about updating this file.

Also can you add a big comment like "coupling: if you modify this, you first need to modify lang.json in the semgrep-langs repository"

Reply:
Yea I can do both, I didn't realize there was a semgrep-langs :O is there a reason we don't include the repo in the app? [Here](https://github.com/returntocorp/semgrep-app/blob/develop/frontend/src/utils/lang.json) it looks like we have an outdated version that's just been copied in.

Reply:
I dunno. They should use the semgrep-langs submodule (we do in semgrep).